src/piadm.sh

```diff
@@ -655,13 +675,245 @@ ispoolenabled() {
 	return 1
 }
 
+# Routines and variables related specifically to Triton Compute Nodes.
+
+# Data for Triton Compute Node (CN) iPXE.
+TRITON_IPXE_PATH=/opt/smartdc/share/usbkey/contents
+TRITON_IPXE_ETC=${TRITON_IPXE_PATH}/etc
+TRITON_IPXE_BOOT=${TRITON_IPXE_PATH}/boot
+
+initialize_as_CN() {
+	TRITON_CN="yes"
+
+	# To get shellcheck to be quiet, even though load_sdc_config DTRT:
+	CONFIG_sapi_domain=""
+
+	source /lib/sdc/config.sh
+	load_sdc_config
+
+	# Establish the CNAPI default boot Platform Image
+	cnapi_domain=$("${CURL[@]}" http://"${CONFIG_sapi_domain}"/applications?name=sdc | json -Ha metadata.cnapi_domain)
+	CNAPI_DEFAULT_PI=$("${CURL[@]}" http://"${cnapi_domain}"/boot/default | json platform)
+}
```

Comment:
Change to:

```
initialize_as_CN() {                                                            
        TRITON_CN="yes"                                                         
                                                                                
        source /lib/sdc/config.sh                                               
        load_sdc_config                                                         
                                                                                
        # Establish the CNAPI default boot Platform Image                       
        # shellcheck disable=2154                                               
        cnapi_domain=$("${CURL[@]}" http://"${CONFIG_sapi_domain}"/applications?name=sdc | json -Ha metadata.cnapi_domain)
        CNAPI_DEFAULT_PI=$("${CURL[@]}" http://"${cnapi_domain}"/boot/default | json platform)
}                                                                               
```

Reply:
Oh damn, you edited this.  Lemme try again...

Reply:
... there we go.